src/cluster_legacy.c

```diff
@@ -546,7 +546,8 @@ int clusterLoadConfig(char *filename) {
             } else if (!strcasecmp(s, "noflags")) {
                 /* nothing to do */
             } else {
-                serverPanic("Unknown flag in redis cluster config file");
+                serverPanic("Unknown flag in %s cluster config file",
+                    server.extended_redis_compat ? "redis" : "valkey");
```

Comment:
This config has limited scope and in general not used for logging. We want to delete it in the future so we don't want to add it everywhere. Please follow #207.

Reply:
Added SERVER_NAME as it smention is #207.